### DISCUSSION TITLE
Can i use Open Source Edition with a Redis Cluster?

### DISCUSSION THREAD

**Discussion by adriangonzalezroble** [redisson/redisson#5567] [Q&A] [answered]
Hi everyone! We are evaluating with my team a possible use of Redisson to integrate our microservices to our Redis Cluster.

Currently, we are using Lettuce for the integration and for now, we only use simple Hash operations (our use case is a simple request cache). But we are looking for features like distributed locks, and Redisson has them natively.

We are using Springboot and Azure Cache for Redis in Cluster mode.

I have doubts due to this comparison of versions https://redisson.pro/#compare-editions 
<img width="1103" alt="Screenshot 2024-01-15 at 09 29 04" src="https://github.com/redisson/redisson/assets/9310576/260f4139-37a2-42fb-b871-a5fde542ca61">

**So, the question is:
Can i use Open Source Edition with a Redis Cluster?**

Thanks in advance for your help!

**Comment by mrniko** [comment, marked as answer]
Open-source is fully compatible with Redis cluster mode. 

The only difference is that each Redisson object will be located on a single master node, while PRO version uses a special algorithm to distribute it's content across all master nodes. Neither data partitioning nor multi cluster features are supported by Lettuce.
